Vagrantfile

```diff
@@ -2,13 +2,20 @@
 # vi: set ft=ruby :
 $script = <<SCRIPT
     cd /vagrant
-    sudo dnf install -y tito python-ipdb python-nose
-    sudo dnf install -y tracer
+    sudo dnf install -y tito tracer
+
+    # the following deps fail during the tito --install due to tito using --cache-only option, so install them beforehand
+    sudo dnf install -y python2-beautifulsoup4 python2-cssselect python2-html5lib python2-rpm python2-webencodings python2-future python2-psutil
```

Comment:
Are you sure this is needed? Since we are doing

    sudo dnf install -y tito tracer

I think the 

    tito build --rpm --test --install

should then work because the dependencies should be already installed when installing tracer from the Fedora repository.

Reply:
Without it there's errors like this when tito installs the built packages:

```
    default: Package "python2-future-0.18.2-3.fc30.noarch" from repository "updates" has incorrect checksum
    default: Error opening file for checksum: /var/cache/dnf/updates-d6521d96a82a4d6a/packages/python2-psutil-5.6.7-1.fc30.x86_64.rpm
    default: Package "python2-psutil-5.6.7-1.fc30.x86_64" from repository "updates" has incorrect checksum
    default: Error opening file for checksum: /var/cache/dnf/fedora-a924e206df91842b/packages/python2-beautifulsoup4-4.6.3-2.fc30.noarch.rpm
    default: Package "python2-beautifulsoup4-4.6.3-2.fc30.noarch" from repository "fedora" has incorrect checksum
```

It seems to be because installing `tracer` on F30 installs python3-tracer, but with the RPM build we get python3-tracer and python2-tracer, and we're missing deps for the latter

Reply:
Oh, okay. This tito issue bothers me for a long time, so I've finally submitted it https://github.com/rpm-software-management/tito/issues/363 and hopefully, I (or somebody else) will get to fixing it soon.

Let's stick with manually installing dependencies as you did in this PR.

Reply:
Thanks for filing that. Will be nice to have fixed some time :)